### ISSUE TITLE
Portamento Glissandi

### ISSUE THREAD

**Issue by Magnichu** [musescore/MuseScore#18794]
### Your idea

I would like to see the portamento glissando feature for trombone work with MuseSounds. 

### Problem to be solved

On trombone, the portamento glissando option does not work with MuseSounds. Portamento glissandi would create an authentic sound to scores with trombone glissandi.

**Comment by s11141827** [comment]
Yeah this applies to Trombone, Trombones A3, & Bass Trombone which have a slide (same for all the Glissandos for all brass instruments, etc set to Portamento), so another update will be needed.

**Comment by Daniel-R15** [comment]
Came up again in **Portamento playback for Trombone** on the musescore.org forum:  https://musescore.org/en/node/368851
